_includes/head.html

```diff
@@ -11,6 +11,7 @@
   <link rel="shortcut icon" href="/assets/favicon.ico">
   <link rel="stylesheet" href="//code.cdn.mozilla.net/fonts/fira.css">
   <link rel="stylesheet" href="/assets/css/bootstrap.min.css">
+  <link rel="stylesheet" href="/assets/css/datepicker/bootstrap-datepicker3.min.css">
```

Comment:
do I see it correctly that this is the only css import? Then all other files could be deleted (such as standalone, and the non-minified version)